scripts/read_neoprep_log.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+DESC="""EPP used to create csv files for the nanoprep robot"""
+import logging
+import os
+
+from argparse import ArgumentParser
+from genologics.lims import Lims
+from genologics.config import BASEURI,USERNAME,PASSWORD
+from genologics.entities import *
+from genologics.epp import attach_file
+
+def read_log(lims, pid, logfile):
+    logger=setupLog(logfile)
+    pro=Process(lims, id=pid)
+    f=None
+    for out in pro.all_outputs():
+        if out.type == "ResultFile" and out.name == "NeoPrep Output Log File":
+            fid=out.files[0].id
+            f=lims.get_file_contents(id=fid)
```

Comment:
:D could have, but this time it stands for file_contents. I should not use such short variables. give me a minute